spec/intro.dd

```diff
@@ -106,7 +106,7 @@ $(H2 Memory Model)
 
     $(P A memory location can be temporarily transferred from shared to local if
     synchronization is used to prevent any other threads from accessing the memory
-    location during the duration.)
+    location while the transfer is in effect.)
```

Comment:
I think it was meant to be "during the operation".
That sentence is a bit ambiguous to me: If the data held in that memory location has indirection, the transfer needs to be either a deep copy or retain `shared` on the inner types.